pkg/types/types.go

```diff
@@ -76,3 +76,55 @@ type Status struct {
 	// newest node conditions in this field.
 	Conditions []Condition `json:"conditions"`
 }
+
+// Type is the type of the problem.
+type Type string
```

Comment:
I prefer keeping these types in each monitor. :)

Reply:
Will do.